docs/design/About.md

```diff
@@ -31,3 +31,7 @@ If you want to know whole system architecture of this framework, please read [th
 ## TierIV Scenario Format Version 2.0
 
 If you want to know TierIV Scenario format Version 2.0 , which is a default scenario format of this framework, please read [this documentation](TierIVScenarioFormatVersion2.md).
+
+## Vehicle Model
+
+If your want to know about vehicle dynamics model in simulation, please check [this documentation](VehicleDynamics.md).
```

Comment:
```suggestion
If you want to know about vehicle dynamics model in simulation, please check [this documentation](VehicleDynamics.md).
```

Reply:
Tank you!
I fixed at https://github.com/tier4/scenario_simulator_v2/pull/540/commits/5a14e79106e83816def95bdecd4892dcdd08e81d